Lint-Whoppers/nutshell
# Mj fix buttons

# Description

reattached event listeners to buttons

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1) `git fetch --all`
2) `git checkout mj-fixButtons`
3)  What to check for buttons working after you edit


# Checklist:
